### PR TITLE
Documentation on Deploying Jobs with Akri

### PR DESCRIPTION
**Pull request by kate-goldenring**
Documentation for changes in https://github.com/project-akri/akri/pull/437

### CODE REVIEW

File: docs/development/debugging.md [hunk 1]
```diff
@@ -38,6 +41,49 @@ helm install akri akri-helm-charts/akri \
...
+
+```bash
+watch kubectl get pods,akric,akrii,services -o wide
+```
+
+Set `debugEcho.configuration.shared=true` to discover Debug Echo devices that are shared by all nodes. For example, when Akri is installed like above with `debugEcho.configuration.shared=false` onto a 3 node cluster. 6 Debug Echo devices will be discovered and 6 Instances will be created, 2 for each Node. However, if `debugEcho.configuration.shared=true` is set, only 2 will be discovered as it is mocking all 3 nodes "utilizing" the same two devices. Set `debugEcho.configuration.capacity=3` to allow all 3 nodes to receive brokers to utilize each of the shared devices. It defaults to `1`.
+
+### Installation with Pod Brokers
```
Comment (jiria) [inline docs/development/debugging.md:58]: Job Brokers?

File: docs/discovery-handlers/onvif.md [hunk 2]
```diff
@@ -38,12 +38,38 @@ Discovery Handlers are passed discovery details that are set in a Configuration
...
+
+### Broker Job Settings
+
+If you would like terminating [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/) to be deployed automatically to discovered cameras, a broker image should be specified (under `brokerJob`) in the Configuration.
+
+> Note only a `brokerJob` OR `brokerPod` should be specified.
+
+| Helm Key | Value | Default | Description |
+| :--- | :--- | :--- | :--- |
+| onvif.configuration.brokerJob.image.repository | image string | "" | image of broker Pod that should be deployed to discovered devices |
```
Comment (romoh) [inline docs/discovery-handlers/onvif.md:62]: image of broker "job"
same below.

File: docs/discovery-handlers/opc-ua.md [hunk 3]
```diff
@@ -28,12 +28,36 @@ The generic OPC UA Configuration takes in a list of DiscoveryURLs, whether for L
...
+| opcua.configuration.brokerPod.resources.memoryLimit | string | "200Mi" | the maximum amount of RAM this Pod can consume. Default based on the Akri OPC UA sample broker. Adjust to the size of your broker. |
+| opcua.configuration.brokerPod.resources.cpuLimit | string | "30m" | the maximum amount of CPU this Pod can consume. Default based on the Akri OPC UA sample broker. Adjust to the size of your broker. |
+
+### Broker Job Settings
+
+If you would like terminating [Jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/) to be deployed automatically to discovered servers, a broker image should be specified (under `brokerJob`) in the Configuration.
+
+> Note only a `brokerJob` OR `brokerPod` should be specified.
+
+| Helm Key | Value | Default | Description |
```
Comment (romoh) [inline docs/discovery-handlers/opc-ua.md:50]: You skipped the image repo and tag here for opc-ua

**Review by romoh — APPROVED** [review]
Left some suggestions

**Pull request merged**